### ISSUE TITLE
can the back button be assigned to any button?

### ISSUE THREAD

**Issue by iamapie13** [polygraphene/ALVR#44]
other gear vr apps can utilize the back button, can it do something for steam vr aswell instead of closing the app?

**Comment by polygraphene** [comment]
Yes. I will add the assignment of back button in near future. But Gear VR apps can only get "short press" of back button, so we can't do long press of SteamVR buttons by back button.

> also, I have a positional tracking solution for my headset and I noticed that the controller is stuck in one position, can the controller follow the headset's position?

If you are using ALVR client ARCore version, I will fix the bug on the next release.

**Comment by chapeka** [comment]
perfect back button like grip, and may be possible buttons combined

**Comment by Muhlawor** [comment]
Would love to remap back button to menu button
